### ISSUE TITLE
Updating Settlements

### ISSUE THREAD

**Issue by erian-7** [BernhardPosselt/pf2e-kingmaker-tools#16]
I know it's alpha, so maybe not functional yet and as such this may not be an issue...

For the Settlement added through the Kingdom Sheet, is there functionality yet to update the associated information? When I Open the settlement the summary pops up but all fields are static. It doesn't seem to account for buildings in the scene (for example adding Houses doesn't affect Overcrowding, having a Town Hall doesn't increase Leadership Activities, etc.).

Again, I know this is alpha content so this may just still be in development.

Thanks!

**Comment by BernhardPosselt** [comment]
That should all work and be read directly from the scene. The popup isn't re-rendered if you change the settlement though, but the kingdom sheet is. If you reopen the settlement info after changing the scene, it should have the up to date info.

**Comment by BernhardPosselt** [comment]
The structure properties are persisted as flags. Maybe you have created the structures manually and did not import them from the compendium?

**Comment by erian-7** [comment]
Ah, it could definitely be old structures since I've been using this from version 0.0.3. I'll also try deleting all the Structure actors then and reimporting them.

**Comment by erian-7** [comment]
Yep, that did it. Deleting the Structures folder and re-importing got it fixed.

Thanks for the save--I was contemplating the pain of rebuilding the world if I couldn't figure this out!
